domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -282,7 +286,11 @@ private ComplexFieldsMapper() {
         }
 
         private static List<CaseField> getCaseFields(List<ComplexFieldEntity> complexFieldEntityList) {
-            return complexFieldEntityList.stream().map(complexFieldEntity -> EntityToResponseDTOMapper.INSTANCE.map(complexFieldEntity)).collect(Collectors.toList());
+            List<CaseField> caseFields = complexFieldEntityList.stream()
+                                                               .map(complexFieldEntity -> EntityToResponseDTOMapper.INSTANCE.map(complexFieldEntity))
+                                                               .collect(Collectors.toList());
```

Comment:
Could be simplified with
.sorted(NULLS_LAST_COMPARATOR)

Reply:
Good point.

Reply:
Done